projects/uitdatabank/docs/entry-api/events/status.md

```diff
@@ -1,8 +1,141 @@
-Coming soon.
-
-<!-- 
-  @todo 
-  Explain how to postpone/cancel an event, and optionally a reason.
-  Maybe show screenshot(s) of how postponed/cancelled events look on UiTinVlaanderen (as an example).
-  Also mention that a new event should be created for the new date(s) when postponed.
--->
+# Cancelling or postponing an event
+
+When [creating](./create.md) or (usually) when [updating](./update.md) your event, you can mark it as postponed or cancelled to indicate to attendees that it will be moved to another date or not take place at all.
+
+Cancelled events automatically get an *"(geannuleerd)"* label next to their calendar info in online calendars. For example, on UiTinVlaanderen:
+
+<!-- focus: false -->
+
+![Screenshot of a summary of the event "Boshyacintenfestival" on UiTinVlaanderen, as an example of the "(Geannuleerd)" label](../../../assets/images/event-cancelled.png)
```

Comment:
I think it is better to use an image / example 
* that has only one status (so not in the title and no duplicate status by UiTinVlaanderen)
* that has a status reason

Example:
![Screenshot 2023-02-10 at 13 13 09](https://user-images.githubusercontent.com/29090370/218090462-130b1959-74b3-4b33-94eb-ffd6f17b5d37.png)

(The copy underneath the image is still relevant)

Reply:
Okay, can you send me a link to that event? Because the screenshot seems to be of a low resolution (looking on a Macbook Pro's built-in screen).

Reply:
Here it is: https://www.uitinvlaanderen.be/agenda/e/wereldkeuken/ac844f27-50f1-4867-be9a-c5bad2538e47

Reply:
I used the screenshot from your comment above, because the tooltip is now displaying very small ([UIT-1696](https://jira.uitdatabank.be/browse/UIT-1696))